File: Modul3/Python/PRAK304_2310817210013_MuhammadAufaFtrianda.py
nilai = int(input("Masukan Angka Anda : "))

if nilai == 0:
    print("")
    print("Nol")
elif 1 <= nilai <= 9:
    print("")
    print("Satuan")
elif 10 <= nilai <= 19:
    print("")
    print("Belasan")
elif 20 <= nilai <= 99:
    print("")
    print("Puluhan")
else:
    print("")
    print("Anda Menginput Melebihi Limit")

File: Modul3/C/PRAK304_2310817210013_MuhammadAufaFtrianda.c
#include <stdio.h>

int main() {

    int angka;
    printf("Masukan Angka Anda\t: ");
    scanf("%d",&angka);

    if (angka == 0)
    {
        printf("Nol");
    }
    else if (angka>=1 && angka<10)
    {
        printf("Satuan");
    }
    else if (angka>10 && angka<20)
    {
        printf("Belasan");
    }
    else if (angka==10)
    {
        printf("Puluhan");
    }
    else if (angka>19 && angka<100)
    {
        printf("Puluhan");
    }
    else if (angka>99)
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
return 0;
}